README.md

```diff
@@ -68,6 +68,11 @@ defmodule MyApp.Endpoint do
 end
 ```
 
+Wildcards can be used in the origins specification, by using regex:
```

Comment:
I'd rephrase this as "Regular expressions can be used in the `:origins` list as well in order to match hosts based on a pattern"